ocean_lib/common/utils/utilities.py

```diff
@@ -73,3 +75,13 @@ def checksum(seed):
 def get_timestamp():
     """Return the current system timestamp."""
     return f"{datetime.utcnow().replace(microsecond=0).isoformat()}Z"
+
+
+def simplify_credential_to_address(credential):
```

Comment:
This function does one thing: takes the "value" out of the credential object :)

Reply:
Based on your assertion that this method takes the "value" out of the credential object, I'd argue that it shouldn't exist because it should be as simple as `credential["value"]`

But if you insist on keeping this method, then I think it should be moved to `common/agreements/consumable.py` so that all the inputs and outputs of credential-related logic can be stored together. I don't think it belongs in `utilities.py` because the rest of the file focuses on id creation, converting web3 instances to blobs of bytes or hex, checksums, and timestamps.

Based on our discussion earlier today, `consumable.py` is separate from the credential-related logic so that users don't need to import the whole `AddressCredential` or `DDO` classes. I'm fine with this.

Reply:
I'll mark this resolved because I see `simplify_credential_to_address` was moved to `ocean_lib/common/ddo/credentials.py`